shell/browser/native_window_views.cc

```diff
@@ -217,7 +219,7 @@ NativeWindowViews::NativeWindowViews(const gin_helper::Dictionary& options,
 #endif
 
   widget()->Init(std::move(params));
-#if defined(OS_WIN)
+#if defined(OS_WIN) || defined(OS_LINUX)
```

Comment:
This change looks like it might have effects beyond the description of the PR—does this fix a bug about resizability on Linux by any chance?

Reply:
Hmm that's true, this would in theory affect `resizable: true` on any FramelessView, right? I can't find any bug reports on that though, maybe it's just a rarely-used combination?